glue/bin/run-egmde

```diff
@@ -5,7 +5,7 @@ if [ "$(id -u)" != 0 ] && [ -z "$DISPLAY" ] && ! snapctl is-connected login-sess
   echo "ERROR: You need one of: root, a DISPLAY, or to connect the login-session-control interface."
   echo ""
   echo "To connect the login-session-control interface:"
-  echo "  sudo snap connect $SNAP_NAME:login-session-control :login-session-control"
+  echo "  sudo snap connect $SNAP_INSTANCE_NAME:login-session-control :login-session-control"
   echo ""
   echo "With the login-session-control interface connected, you can also add to login shells:"
   echo "  sudo cp /snap/egmde-confined-desktop/current/usr/share/wayland-sessions/egmde-confined-desktop.desktop /usr/share/wayland-sessions/"
```

Comment:
This comment isn't parallel-install aware. But neither is `parallel-installable`